packages/main/src/themes/sap_horizon_dark/Token-parameters.css

```diff
@@ -6,10 +6,18 @@
 	--_ui5_token_selected_focus_outline: none;
 	--_ui5_token_focus_outline: none;
 	--_ui5_token_outline_offset: 0.125rem;
-	--_ui5_token_selected_hover_background: var(--sapButton_Selected_Background);
+	--_ui5_token_selected_hover_background: var(--sapButton_Selected_Hover_Background);
 	--_ui5_token_selected_hover_border_color: var(--sapButton_Selected_BorderColor);
 	--ui5_token_focus_pseudo_element_content: '';
-	--_ui5_token_focus_outline_border_radius: 0.625rem;
+	--_ui5_token_icon_padding: 0.25rem 0.5rem;
+	--_ui5_token_border_radius: 0.375rem;
+	--_ui5_token_focus_outline_border_radius: .5rem;
+	--_ui5_token_text_color: var(--sapTextColor);
```

Comment:
Isn't --_ui5_token_text_color: var(--sapTextColor); valid also for the HC themes. I do not see delta specified also per my analysis it resolves to the same value as the current one applied, so could you please check or ask the visual designers?